prow/cluster/README.md

```diff
@@ -11,6 +11,7 @@ This folder contains files related to the configuration of the Prow production c
 The structure of the folder looks as follows:
 
 ```
+  ├── 00-deck-patch.yaml                # Patch file for the deck deployment used to enable HTTP->HTTPS redirection. 	
```

Comment:
```suggestion
  ├── 00-deck-patch.yaml                # The patch file for the Deck Deployment used to enable HTTP to HTTPS redirection. 	
```

Reply:
`00`? :)

Reply:
Yes, this scripts should be run first, as it modifies the prow installation deployments, so I could either mark it as `00` or as `01` and bump the rest of scripts by 1 :)

Reply:
Yup, I was just wondering why you chose the first option ;)